Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -40,27 +41,21 @@ protocol TabCellDelegate: class {
 }
 
 class TabCell: UICollectionViewCell {
-    enum Style {
-        case light
-        case dark
-    }
-
     static let Identifier = "TabCellIdentifier"
     static let BorderWidth: CGFloat = 3
 
-    var style: Style = .light {
-        didSet {
-            applyStyle(style)
-        }
-    }
-
     let backgroundHolder = UIView()
     let screenshotView = UIImageViewAligned()
+    let titleBackgroundView = GradientView(
+        colors: [UIColor(white: 1.0, alpha: 0.98), UIColor(white: 1.0, alpha: 0.9), UIColor(white: 1.0, alpha: 0.0)],
```

Comment:
Not necessary but last one could just be `.white`

Reply:
Can't be `.white`. specifically because it has 0 alpha

Reply:
`.clear`? Read alpha wrong.

Reply:
Unfortunately `.clear` is white: 0, alpha: 0 which gives makes the final part of the gradient move to 0 alpha black instead of white

Reply:
Ugh, fine, haha :D